engine/src/main/java/org/terasology/rendering/assets/shader/ShaderProgramFeature.java

```diff
@@ -21,11 +21,11 @@
 /**
  */
 public enum ShaderProgramFeature {
-    FEATURE_REFRACTIVE_PASS (0b00000001),
-    FEATURE_ALPHA_REJECT (0b00000010),
-    FEATURE_LIGHT_POINT (0b00000100),
-    FEATURE_LIGHT_DIRECTIONAL (0b00001000),
-    FEATURE_USE_MATRIX_STACK (0b00100000),
+    FEATURE_REFRACTIVE_PASS      (0b00000001),
+    FEATURE_ALPHA_REJECT         (0b00000010),
+    FEATURE_LIGHT_POINT          (0b00000100),
+    FEATURE_LIGHT_DIRECTIONAL    (0b00001000),
+    FEATURE_USE_MATRIX_STACK     (0b00100000),
```

Comment:
@Cervator: can you double check on this? Is it ok to add some spaces this way? We discussed these kind of things once and in general it wasn't a good idea to add formatting spaces. Is this a good exception to the rule?